### ISSUE TITLE
As regards the regularizer on hypergraph

### ISSUE THREAD

**Issue by 0x10C** [Yue-Group/HGNN#17]
As written in paper ,  \Omega(f) is a regularize on hypergraph. I  reviewed your implement code, only found emprirical loss in your objective function.  Is it necessary add this regularizer into loss function? I 'd appreciate if you can answer ASAP, thanks !

**Comment by yifanfeng97** [comment]
Hi, the \Omega(f) is defined in traditional hypergraph learning. In our HGNN, we remove the \Omega(f) in the loss function, and it is transformed as the hypergraph convolution.

**Comment by yifanfeng97** [comment]
More details of the hypergraph neural network can be found in https://github.com/iMoonLab/DeepHypergraph/ .

**Comment by 0x10C** [comment]
Got it. thx
